src/routes/safe/components/Apps/components/AppCard/index.tsx

```diff
@@ -125,15 +146,37 @@ const AppCard = ({ app, iconSize = 'md', to, onPin, onRemove, pinned = false }:
     )
   }
 
+  const shareSafeAppLabel = `Share ${app.name} Safe App`
+
   const content = (
     <>
+      <LeftIconBtn
+        aria-label={shareSafeAppLabel}
+        title={shareSafeAppLabel}
+        onClick={(e) => {
+          // prevent triggering the link event
+          e.preventDefault()
```

Comment:
This can be removed I think as its not a link element thats being rendered but a button.

Reply:
Buttons are also good to prevent, some might submit forms if they happen to be in one.

Reply:
yep, I will keep it, because if I remove it, the browser navigates to the Safe App when you click on the button (same for OnPin & OnRemove buttons in the same file)

Reply:
You might also add `e.stopPropagation()` then, so that it doesn't even reach the link.